### DISCUSSION TITLE
[blocks]: Multi-step form

### DISCUSSION THREAD

**Discussion by sspoth** [shadcn-ui/ui#3263] [Blocks Request]
### Description

A page for submitting a form where you multi steps, a button to move to the next section, or back to a previous section.

**Comment by alexander-densley** [comment]
yes please

**Comment by GenerationUX** [comment]
A Stepper component, like these? (Row, and Column instances, showing variants with 2-5 steps)

![StepperColumn](https://github.com/shadcn-ui/ui/assets/16335131/519e19e1-665e-4b0d-b542-1ddc61106df9)
![StepperRow](https://github.com/shadcn-ui/ui/assets/16335131/16814239-8497-4fd9-8f77-d0d9f051d0bd)

**Comment by anishshobithps** [comment]
@sspoth @alexander-densley @cf62   there is a [PR for Stepper function](https://github.com/shadcn-ui/ui/pull/318),

Also there are [some great examples](https://github.com/damianricobelli/shadcn-stepper/tree/main/examples) you can find by the author of the PR, it has w.r.t to Form as well.
